src/appshell/view/dockwindow_kdab/internal/indicatorswindow.cpp

```diff
@@ -0,0 +1,174 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "indicatorswindow.h"
+
+#include "dropindicators.h"
+
+#include "thirdparty/KDDockWidgets/src/private/Utils_p.h"
+
+#include <QQmlContext>
+
+using namespace mu::dock;
+
+IndicatorsWindow::IndicatorsWindow(DropIndicators* dropIndicators)
+    : QQuickView(),
+    m_dropIndicators(dropIndicators)
+{
+    setFlags(flags() | Qt::FramelessWindowHint | Qt::BypassWindowManagerHint | Qt::Tool);
+    setColor(Qt::transparent);
+
+    rootContext()->setContextProperty(QStringLiteral("_window"), QVariant::fromValue<QObject*>(this));
```

Comment:
I think better rename it to "dockDropIndicator"

Reply:
renamed _window to dockDropIndicatorsWindow